adaptor/adaptor.go

```diff
@@ -95,6 +110,16 @@ func (b Block[BP]) Accept(ctx context.Context) error { return b.vm.AcceptBlock(c
 // Reject calls RejectBlock(b) on the [ChainVM] that created b.
 func (b Block[BP]) Reject(ctx context.Context) error { return b.vm.RejectBlock(ctx, b.b) }
 
+// ShouldVerifyWithContext calls ShouldVerifyWithContext(ctx, b.b) on the [ChainVM] that created b.
+func (b Block[BP]) ShouldVerifyWithContext(ctx context.Context) (bool, error) {
+	return b.vm.ShouldVerifyWithContext(ctx, b.b)
+}
+
+// VerifyWithContext calls VerifyWithContext(ctx, b.b) on the [ChainVM] that created b.
```

Comment:
```suggestion
// ShouldVerifyWithContext calls ShouldVerifyWithContext(ctx, b) on the [ChainVM] that created b.
func (b Block[BP]) ShouldVerifyWithContext(ctx context.Context) (bool, error) {
	return b.vm.ShouldVerifyWithContext(ctx, b.b)
}

// VerifyWithContext calls VerifyWithContext(ctx, b) on the [ChainVM] that created b.
```

Although this makes the documentation not _strictly_ true, the internal `.b` field is an implementation detail of which the consumer is unaware (it may also change). The goal, as with the other similar comments, is to communicate behaviour and effect to the consumer.